src/test/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/ActionSvcClientImplTest.java

```diff
@@ -0,0 +1,132 @@
+package uk.gov.ons.ctp.response.collection.exercise.service.impl;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponents;
+import org.springframework.web.util.UriComponentsBuilder;
+import uk.gov.ons.ctp.common.rest.RestUtility;
+import uk.gov.ons.ctp.response.action.representation.ActionPlanDTO;
+import uk.gov.ons.ctp.response.collection.exercise.client.impl.ActionSvcRestClientImpl;
+import uk.gov.ons.ctp.response.collection.exercise.config.ActionSvc;
+import uk.gov.ons.ctp.response.collection.exercise.config.AppConfig;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ActionSvcClientImplTest {
+
+    private static final String ACTION_PATH = "/actions";
+    private static final String HTTP = "http";
+    private static final String LOCALHOST = "localhost";
+    private static final String ACTION_PLAN_NAME = "example";
+    private static final String ACTION_PLAN_DESCRIPTION = "example description";
+
+    @Mock
+    private AppConfig appConfig;
+
+    @InjectMocks
+    private ActionSvcRestClientImpl actionSvcClient;
+
+    @Mock
+    private RestTemplate restTemplate;
+
+    @Mock
+    private RestUtility restUtility;
+
+    @Before
+    public void before() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
I don't think you need this line if your using `@RunWith(MockitoJUnitRunner.class)`